components/TextFields/tests/snapshot/MDCBaseTextFieldSnapshotTests.m

```diff
@@ -18,6 +18,8 @@
 
 #import "MaterialTextFields+ContainedInputView.h"
 
+static const NSTimeInterval kTextFieldValidationAnimationTimeout = 1.5;
```

Comment:
1.5 second timeout seems like a large value. Did lower values not work? something closer to .1 or even lower is what i would expect. It only seems to need to pump the run loop once to render the view.
Are you expecting an animation to occur?

Reply:
I made it 1 just now in 35b1fb59ea6828b2fbb9ebb17a9b95894a7da2d0, is that low enough? To be clear, this is just how long the expectation waits before failing, the tests run much much faster than that. I picked a high-ish number out of fear of random delays leading to flaky tests. I'm sort of afraid of flaky tests 😅